source/routing/tests/routing_api_test.cpp

```diff
@@ -3378,3 +3379,434 @@ BOOST_FIXTURE_TEST_CASE(should_filter_late_journeys_with_empty_list, Night_bus_f
+    // clockwise = true
+    // min_nb_journeys = 2
+    // timeframe_duration = 40*60 (40min)
+    // timeframe_max_bound = 1H
+    //
+    // the reponse should contain at least 2 journeys
+    min_nb_journeys = 2;
+    timeframe_duration = 40*60; // 40 min
+    clockwise = true;
+    timeframe_max_duration = 60*60;
+
+    // send request
+    navitia::PbCreator pb_creator_case6(data_ptr, "20180309T075900"_dt, null_time_period);
+    make_response(pb_creator_case6,
+                  raptor,
+                  origin,
+                  destination,
+                  {ntest::to_posix_timestamp("20180309T075900")},
+                  clockwise,
+                  navitia::type::AccessibiliteParams(),
+                  forbidden,
+                  {},
+                  sn_worker,
+                  nt::RTLevel::Base,
+                  2_min,
+                  timeframe_max_duration,
+                  10,
+                  0,
+                  0,
+                  0,
+                  boost::none,
+                  1.5,
+                  900,
+                  timeframe_duration);
+
+    // get the response
+    resp = pb_creator_case6.get_response();
+    BOOST_REQUIRE_EQUAL(resp.response_type(), pbnavitia::ITINERARY_FOUND);
+    BOOST_REQUIRE_GE(resp.journeys_size(), 2);
+
+    //-----------------------------------
+    // Case 7 :
+    // clockwise = true
+    // min_nb_journeys = 10
+    // timeframe_duration = 1 day
+    // timeframe_max_bound = 10 min
+    //
+    // We test the case where max_duraion < timeframe_duration
+    // Even though we ask for 10 journeys, the search bound is limited to 10min
+    // Raptor can find only one solution within ten minutes
+    min_nb_journeys = 10;
+    timeframe_duration = 24*60*60; // 1 day
+    clockwise = true;
+    timeframe_max_duration = 10*60; // 10min
+
+    // send request
+    navitia::PbCreator pb_creator_case7(data_ptr, "20180309T075900"_dt, null_time_period);
+    make_response(pb_creator_case7,
+                  raptor,
+                  origin,
+                  destination,
+                  {ntest::to_posix_timestamp("20180309T075900")},
+                  clockwise,
+                  navitia::type::AccessibiliteParams(),
+                  forbidden,
+                  {},
+                  sn_worker,
+                  nt::RTLevel::Base,
+                  2_min,
+                  timeframe_max_duration,
+                  10,
+                  0,
+                  0,
+                  0,
+                  boost::none,
+                  1.5,
+                  900,
+                  timeframe_duration);
+
+    // get the response
+    resp = pb_creator_case7.get_response();
+    BOOST_REQUIRE_EQUAL(resp.response_type(), pbnavitia::ITINERARY_FOUND);
+    BOOST_REQUIRE_EQUAL(resp.journeys_size(), 1);
+
+    //-----------------------------------
+    // Case 8 :
+    // clockwise = true
+    // min_nb_journeys = 5
+    // timeframe_duration = 1 day
```

Comment:
this line is describing the same param than 2 lines below but saying different things